dev-tools/mage/gotest.go

```diff
@@ -94,6 +118,33 @@ func DefaultTestBinaryArgs() TestBinaryArgs {
 	}
 }
 
+// GoTestIntegrationForModule executes the Go integration tests.
+// Use TEST_COVERAGE=true to enable code coverage profiling.
+// Use RACE_DETECTOR=true to enable the race detector.
+func GoTestIntegrationForModule(ctx context.Context) error {
+	return RunIntegTest("goIntegTest", func() error {
+		modulesFileInfo, err := ioutil.ReadDir("./module")
```

Comment:
Not all beats have their "modules" under `module`, maybe the modules path should be parameterized. But we can change this in a future change if we find other beats needing this functionality.

Reply:
I think we may come up to this in the future, right now only Metricbeat (and x-pack Metricbeat) uses this feature.